users/templates/users/user_edit.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="row">
+
+        <div class="col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 col-lg-4 col-lg-offset-4">
+
+            <h2>Edit your profile</h2>
+
+            {% if form.errors %}
+                <div class="alert alert-danger" role="alert">There was an error.</div>
+            {% endif %}
+
+            <form action="" method="post">
+                {% csrf_token %}
+
+                <br/>
+                <input type="text"
+                       name="username"
+                       class="form-control floating-label {% if 'username' in form.errors %} has-error{% endif %}"
+                       placeholder="Username"
+					   value="{{ user.username}}"
```

Comment:
pyCharm did its formatting, but as it doesn't display spaces/tabs, I never noticed that it silently mixed them.